.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
 - mkdir -p ../freesound-data/uploads
 - mkdir -p ../freesound-data/avatars
 install:
+- pip install -U pip
```

Comment:
Yes, that will be a problem when merging. But if I don't add this it will fail on build. 
Any suggestions?